cosmic/tests/test_sample.py

```diff
@@ -9,33 +9,73 @@
 import pandas as pd
 from cosmic.sample.sampler.independent import Sample
 from cosmic.sample.sampler.multidim import MultiDim
-
+from scipy.optimize import curve_fit
+from cosmic.utils import a_from_p
 
 SAMPLECLASS = Sample()
 MULTIDIMSAMPLECLASS = MultiDim()
 TEST_DATA_DIR = os.path.join(os.path.split(__file__)[0], 'data')
-TOTAL_SAMPLED_MASS_KROUPA93 = 39.253472063203404
-TOTAL_SAMPLED_MASS_KROUPA01 = 28.708454104214034
-TOTAL_SAMPLED_MASS_SALPETER55 = 22.575833208553195
-TOTAL_SECONDARY_MASS = 16.41457676168502
+KROUPA_93_HI = -2.7
+KROUPA_93_MID = -2.2
+KROUPA_93_LO = -1.3
+KROUPA_01_HI = -2.3
+KROUPA_01_LO = -1.3
+SALPETER_55 = -2.35
 N_BINARY_SELECT = 85
 VANHAAFTEN_BINFRAC_MAX = 0.9989087986493874
 VANHAAFTEN_BINFRAC_MIN = 0.6192803136799157
 MULTIDIM_BINFRAC_MAX = 0.6146916774140262
 MULTIDIM_BINFRAC_MIN = 0.13786300908773025
 PORB = 670.88711347
 PORB_RENZO19 = 303.789477175485
-PORB_SANA12 = 43.52635793101753
-THERMAL_ECC_SUM = 5.7488819291685695
-UNIFORM_ECC_SUM = 3.58801017672414
-SANA12_ECC_SUM = 0.03346470981696298
+SANA12_PORB_POWER_LAW = -0.55
+FLAT_SLOPE = 0.0
+THERMAL_SLOPE = 2.0
+SANA12_ECC_POWER_LAW = -0.42
```

Comment:
We should either stick the best-fit parent distribution numbers (porb:-0.55, ecc:-0.45) or the mean of the retrieved  parameters (porb:-0.38, ecc=-0.42). But not mix and match. 

I'd recommend the first set.